src/test/java/uk/gov/hmcts/cmc/claimstore/services/notifications/CCJRequestedNotificationServiceTest.java

```diff
@@ -42,7 +43,15 @@ public void setup() {
 
     @Test
     public void notifyClaimantShouldCallNotify() throws Exception {
-        Claim claim = SampleClaim.getDefault();
+        Claim claim = SampleClaim.builder()
```

Comment:
why?

Reply:
need ccj requested date which is not there in default claim.